crates/sui-core/src/unit_tests/narwhal_manager_tests.rs

```diff
@@ -160,16 +160,16 @@ async fn test_narwhal_manager() {
 
         let narwhal_manager = NarwhalManager::new(narwhal_config, metrics);
 
-        let consensus_handler_initializer = ConsensusHandlerInitializer {
-            state: state.clone(),
-            checkpoint_service: checkpoint_service_for_testing(state.clone()),
-            epoch_store: epoch_store.clone(),
-            low_scoring_authorities: Arc::new(Default::default()),
-            throughput_calculator: Arc::new(ConsensusThroughputCalculator::new(
+        let consensus_handler_initializer = ConsensusHandlerInitializer::new(
+            state.clone(),
+            checkpoint_service_for_testing(state.clone()),
+            epoch_store.clone(),
+            Arc::new(Default::default()),
```

Comment:
You can probably now simplify some of these params

Reply:
Oh like for instance you don't even need to pass `Arc::new(Default::default())` here. Basically you could pass whatever is the minimum amount of information needed by the initializer, which then can construct more complex data structures in the constructor (e.g. clone, Arc and etc.)

Reply:
Ah got it! Yes probably for the testing purposes I can do some reduction - maybe via a separate method - for the real one I think we are to the minimum needed as most of the injected objects are already Arcs etc. Let me update.